### PR TITLE
DDF-3013 Update security expansion commands to use annotations instead of blueprint.

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Switches the security expansion commands to use the new annotated method of creating commands instead of the deprecated gogo way.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@vinamartin @emmberk @josephthweatt
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).

[Security](https://github.com/orgs/codice/teams/security)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@figliold
#### How should this be tested? (List steps with links to updated documentation)
Verify build passes. 
#### What are the relevant tickets?
[DDF-3013](https://codice.atlassian.net/browse/DDF-3013)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

File: platform/security/expansion/security-expansion-commands/pom.xml [hunk 1]
```diff
@@ -50,11 +50,11 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
+                        <Karaf-Commands>*</Karaf-Commands>
```
Comment (emmberk) [inline platform/security/expansion/security-expansion-commands/pom.xml]: Is this line needed? I see other bundles that already have the new karaf command annotations do not ahve this line.

Comment (clockard) [inline platform/security/expansion/security-expansion-commands/pom.xml]: The commands are not registered or available without this. I think there is another way of doing this as well with another build plugin but this seemed simpler.

File: platform/security/expansion/security-expansion-commands/pom.xml [hunk 2]
```diff
@@ -50,11 +50,11 @@
...
                         <Export-Package/>
                         <Import-Package>
                             ddf.security.expansion,
                             org.apache.felix.service.command,
```
Comment (vinamartin) [inline platform/security/expansion/security-expansion-commands/pom.xml]: ❓ Do we still need this?

File: platform/security/expansion/security-expansion-commands/src/main/java/org/codice/ddf/security/command/ExpandCommand.java [hunk 3]
```diff
@@ -27,22 +29,25 @@
  * Implements the "expand" command - taking an attribute name and the current value, and prints out
  * what the expansion of that value is using the currently-configured expansion service.
  */
+
 @Command(scope = "security", name = "expand", description = "Expands a given key and set of values.")
```
Comment (vinamartin) [inline platform/security/expansion/security-expansion-commands/src/main/java/org/codice/ddf/security/command/ExpandCommand.java]: ✏️   Style nitpick: We usually have the `@Service` annotation before `@Command`

File: platform/security/expansion/security-expansion-commands/pom.xml (hunk 2 above)
Comment (clockard) [inline platform/security/expansion/security-expansion-commands/pom.xml]: I don't think so. I will remove it.

File: platform/security/expansion/security-expansion-commands/src/main/java/org/codice/ddf/security/command/ExpandCommand.java (hunk 3 above)
Comment (clockard) [inline platform/security/expansion/security-expansion-commands/src/main/java/org/codice/ddf/security/command/ExpandCommand.java]: I can change that to be consistent.

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
Manual Testing passed

**Pull request merged**